js/manager.js

```diff
@@ -38,7 +38,7 @@ export const DFPManager = Object.assign(new EventEmitter(), {
           });
           const targetingArguments = this.getTargetingArguments();
           Object.keys(targetingArguments).forEach((varName) => {
-            if (targetingArguments.hasOwnProperty(varName)) {
+            if (targetingArguments[varName]) {
```

Comment:
this is good catch!, no need of calling ``` hasOwnProperty()``` at all!, BUT on the other hand, we are introducing a bug/restriction, explanation: we should also allow null, undefined, '' or 0 values to get passed through. Let's just delete the if stanza:
```
Object.keys(targetingArguments).forEach((varName) => {
    pubadsService.setTargeting(varName, targetingArguments[varName]);
});
```